### PR TITLE
RPresto

### PR DESCRIPTION
**Pull request by RyanHackney**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/r-rpresto/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    - r-httr >=0.6
+    - r-jsonlite
+    - r-openssl
+    - r-stringi
+
+  run:
+    - r-base
+    - {{native}}gcc-libs         
+    - r-dbi >=0.3.0
+    - r-rcpp >=0.12.7
```
Comment (xhochy) [inline recipes/r-rpresto/meta.yaml:48]: Rcpp is a built time only dependency and should be omitted from run.

File: recipes/r-rpresto/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  summary: 'Implements a ''DBI'' compliant interface to Presto. Presto is an open source distributed
+    SQL query engine for running interactive analytic queries against data sources of
+    all sizes ranging from gigabytes to petabytes: <https://prestodb.io/>.'
+  license_family: BSD
+  license_file: LICENSE.txt
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/BSD_3_clause'
+    - LICENSE
+
+extra:
+  recipe-maintainers:
```
Comment (xhochy) [inline recipes/r-rpresto/meta.yaml:71]: Please add conda-forge/r as a maintainer

**Comment by dbast** [comment]
@RyanHackney Please use https://github.com/bgruening/conda_r_skeleton_helper to create R recipes... see my update commit https://github.com/conda-forge/staged-recipes/pull/12766/commits/6441f5edd591f9e0bf470f22a887becd51e2dfcf

**Comment by dbast** [comment]
@rluria14 @ndmaxar @Oblute Do you agree being maintainers for the recipe?

@RyanHackney Is the update commit clear? That was a bit much to suggest via comments. Next time using https://github.com/bgruening/conda_r_skeleton_helper would be easier.

**Comment by dbast** [comment]
@xhochy r-rcpp is also required as run dependency...

**Comment by xhochy** [comment]
> @xhochy r-rcpp is also required as run dependency... 

Sorry, got this mixed up with r-cpp11 🙈

**Comment by RyanHackney** [comment]
@oblute @ndmaxar @rluria14 agree to be maintainers?

**Pull request merged**